### DISCUSSION TITLE
API > Be able to get the followers of any given organization

### DISCUSSION THREAD

**Discussion by adriens** [forem/forem#18247] [Feature Requests]
# :grey_question: Context

It is actually possible to : 

- Get the [followers of the current user](https://developers.forem.com/api/#tag/followers)
- Get the [users of a given organization](https://developers.forem.com/api/#operation/getOrgUsers)
- Get the [articles of a given organization](https://developers.forem.com/api/#operation/getOrgArticles)

:point_up:  ... but we are not able to get the followers of a any given organization.

# :pray: Feature request :bulb: 

:point_right:  For reporting purpose, being able to get the followers of any organization would be really interesting

**Comment by joergrech** [comment]
I need a similar feature: I just want the count of followers in the organization data JSON (e.g., https://dev.to/api/organizations/paydevs)

**Comment by barrymichaeldoyle** [comment]
I would like the API to support getting just the follower count for a the current user, a bonus would be to get the follower count for a specific user, I don't see a downside to making that publicly available.
